.github/scripts/create-release.sh

```diff
@@ -29,7 +29,7 @@ MAJOR_MINOR_VERSION=`echo $REL_VERSION | cut -d. -f1,2`
 RELEASE_BRANCH="release-$MAJOR_MINOR_VERSION"
 RELEASE_TAG="v$REL_VERSION"
 
-if [ `git rev-parse --verify $RELEASE_BRANCH 2>/dev/null` ]; then
+if [ `git rev-parse --verify origin/$RELEASE_BRANCH 2>/dev/null` ]; then
   echo "$RELEASE_BRANCH branch already exists, checking it out ..."
   git checkout $RELEASE_BRANCH
```

Comment:
Do we need `origin/` here too?

Reply:
No, git is smart enough to pull from origin automatically. I cut the release running this script on my desktop after doing a git clone, so brand new checked out code with no local branch.

Reply:
Also, doing that, caused the original error we were seeing in the GH actions run.